modules/ROOT/pages/cache-scope-strategy.adoc

```diff
@@ -3,32 +3,40 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can create a Caching Strategy from a Cache scope or from a Global Elements configuration in Anypoint Studio. Once you create a strategy, a Cache component in your flow can reference it.
+You can create a Caching Strategy from a Cache scope properties panel or a Global Elements configuration in Anypoint Studio. Once you create a strategy, a Cache component in your flow can reference it.
+
+Follow these steps to configure a Caching Strategy:
 
 . Open the Caching Strategy configuration window.
 +
-You can open it from the Cache component configuration tab or from the Caching Strategies option in the Global Elements tab within Studio.
+You can open it from the Cache scope properties panel or the Caching Strategies option in the Global Elements tab within Studio:
++
+image:mruntime-caching-strategy.png[caching strategy configuration]
 +
-. Provide a Name for your Caching Strategy.
-. Reference an ObjectStore or create one for your Cache component to reference.
+. Define the *Name* of the Caching Strategy.
+. Reference an *ObjectStore* or create one for the Cache scope to reference.
 +
 . Select a mechanism for generating a key used for storing events within the Caching Strategy:
 +
-* Default
-* Key Expression: A DataWeave expression (for example, `keyGenerationExpression="#[vars.requestId]"`).
+* *Default*
+* *Key Expression* +
+A DataWeave expression (for example, `keyGenerationExpression="#[vars.requestId]"`).
```

Comment:
Is this line meant to explain what a key expression is? If it is, the plus sign after Key Expression is in the wrong place.